### DISCUSSION TITLE
Pending  transaction

### DISCUSSION THREAD

**Discussion by Mategio14** [Chia-Network/chia-blockchain#17245] [Support]
Hello my dear friends. It's been 10 days I have this pending. issue. Nobody cares about. Please assist me. I need your help folks

**Comment by BrandtH22** [comment]
@Mategio14  , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The chain has seen high utilization leading to fee pressure so any transactions submitted without fees are taking lower priority than those which have fees. Many users have been able to resolve their pending fees by cancelling the previous transaction onchain and resubmitting it with a higher fee. 

We have created this guide of standard transaction types and their minimum effective fees (minimum meaning that time sensitive transactions may require more in fees to be processed accordingly):
https://docs.chia.net/coin-set-costs/#estimated-transaction-costs

Let us know if you have any questions or run into issues when resubmitting the transaction :+1:

**Comment by BrandtH22** [comment]
Closing as this is a duplicate of https://github.com/Chia-Network/chia-blockchain/discussions/17221
